test/recipes/30-test_evp_data/evpkdf_tls13_kdf.txt

```diff
@@ -5024,3 +5024,17 @@ Ctrl.data = hexdata:7c92f68bd5bf3638ea338a6494722e1b44127e1b7e8aad535f2322a644ff
 Ctrl.prefix = hexprefix:746c73313320
 Ctrl.label = hexlabel:6320652074726166666963
 Output = a8464234c7957b85460bf7abda8e20aa43b9e0944c02d76c1c28672619cf6978
+
+# Test that salt of arbitrary length works
+Availablein = fips
+FIPSversion = >=3.4.0
+KDF = TLS13-KDF
+Unapproved = 1
```

Comment:
You'll probably need to remove this line along with removing the Availablein = fips.

Reply:
Why? I am pretty sure TLS13-KDF is available in the FIPS provider, and the key size I used in the test is unapproved.

Reply:
For the Unapproved = 1 check you would need to run this only with the fips provider. But removing this line just ignores the Unapproved indicator value.

Reply:
Removing this cause the fips test to complain about key length, but adding the ctrl to make it ignore key size checks made the default provider complain about an unknown ctrl.
In the end I resulted to provide a key length that passes fips checks, so we avoid any issues in the fips provider and removed all the other options

Reply:
Elsewhere we've split these into two tests.  One with `Availablein = default` and the second with `Availablein = fips`.  Since the two seem to have different behaviour (from what's written here, I've not checked), this probably makes sense to do.